XenAdmin/Alerts/Types/LeafCoalesceAlert.cs

```diff
@@ -28,6 +28,7 @@
  * SUCH DAMAGE.
  */
 
+using System;

```

Comment:
nit but do we need this now?

Reply:
Yes as it is needed for the Action type on line 111